src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -60,36 +74,279 @@ else if (dateArray.length == 3) {
                             calendar.set(year, month - 1, day);
-                            result.setDate(calendar.getTime());
+                            monument.setDate(calendar.getTime());
                         }
                     }
 
                     break;
                 case 6: // Material
-                    result.setMaterial(value);
+                    monument.setMaterial(value);
+                    break;
+                case 7: // Inscription
+                    monument.setInscription(value);
                     break;
                 case 9: // Latitude
                     if (!value.isEmpty()) {
-                        result.setLat(Double.parseDouble(value));
+                        monument.setLat(Double.parseDouble(value));
                     }
                     break;
                 case 10: // Longitude
                     if (!value.isEmpty()) {
-                        result.setLon(Double.parseDouble(value));
+                        monument.setLon(Double.parseDouble(value));
                     }
                     break;
                 case 11: // City
-                    result.setCity(value);
+                    monument.setCity(value);
                     break;
                 case 12: // State
-                    result.setState(value);
+                    monument.setState(value);
                     break;
+                case 13: // Tag columns
+                case 14:
+                case 15:
+                case 16:
+                case 17:
+                case 18:
+                case 20:
+                    if (!value.isEmpty()) {
+                        // Split on commas in-case there are more than one Tag in the column
+                        String[] tagArray = value.split(",");
+
+                        for (int tagArrayColumnIndex = 0; tagArrayColumnIndex < tagArray.length; tagArrayColumnIndex++) {
+                            Tag newTag = new Tag();
+
+                            String tagValue = tagArray[tagArrayColumnIndex];
+                            // Set the first letter of the Tag to upper-case to attempt to reduce duplicates
+                            tagValue = tagValue.strip();
+                            tagValue = tagValue.substring(0, 1).toUpperCase() + tagValue.substring(1);
+                            newTag.setName(tagValue);
+                            newTag.addMonument(monument);
+
+                            result.addTag(newTag);
+                        }
+                    }
+
+                    break;
+                case 22: // Reference
+                    Reference newReference = new Reference();
+                    newReference.setUrl(value);
+                    newReference.setMonument(monument);
+
+                    monument.addReference(newReference);
             }
         }
 
+        result.setMonument(monument);
+
         return result;
     }
 
+    /**
```

Comment:
Instead of just pulling the Monument's description, I decided it would be worth the effort to get a baseline for converting it into an entire CSV row with all of its attributes so we can build on it later when we need to support Monument CSV exports.

Reply:
This is also useful because eventually I will pull all the addresses from google and will need to export that data for everyone else to use 👍